src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/PrisonerSessionValidatorTest.kt

```diff
@@ -30,22 +30,35 @@ class PrisonerSessionValidatorTest {
     private val prisonerDetail3LevelPrison = PrisonerDetailDto("A4000001", unitCode1 = "C", unitCode2 = "1", unitCode3 = "004", unitCode4 = null)
     @Test
     fun `session available to all prisoners when permitted session location is empty`() {
-      val sessionTemplate = createSessionTemplate(permittedSessionLocations = mutableListOf())
+      // Given
```

Comment:
Sorry @dhirajshettymoj I have update ur unit test always like the  //Given//Then//When... it's a standard approach in this code base.. I like it, makes code readable.

Reply:
Thanks! Yes, it's more readable.